freshmaker/config.py

```diff
@@ -370,6 +370,12 @@ class Config(object):
             'default': '',
             'desc': 'A suffix to add to the rebuilt_nvr release in addition to the timestamp.',
         },
+        'previous_rebuilds_autotrigger': {
+            'type': bool,
+            'default': False,
```

Comment:
I'd prefer to enable this by default, and we will not need to update the configurations (`conf/config.py` for locally, or playbooks for our deployments).